libbeat/metric/system/process/process.go

```diff
@@ -402,6 +402,7 @@ func (procStats *Stats) GetOne(pid int) (common.MapStr, error) {
 	}
 
 	e := procStats.getProcessEvent(p)
+	procStats.ProcsMap = newProcs
```

Comment:
I don't fully get why is this updated, could you explain a bit about it? sorry, :innocent:

Reply:
It contains the last samples. It must be updated so based on the previous and the current samples averages and percentages can be calculated.